dell/xps/13-9370/default.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel/kaby-lake
+    ../../../common/pc/laptop
+    ../../../lenovo/thinkpad/acpi_call.nix
+    ../../../lenovo/thinkpad/cpu-throttling-bug.nix
```

Comment:
Hm, interesting. So neither this nor acpi_call are specific to ThinkPads?

Reply:
Correct, but I only played around with acpi_call a while ago, without any specific result. The throttling bug seems to hit most Dell and Thinkpads (and maybe other vendors) models with the latest Kabby Lake CPUs as far as I understand it.

Reply:
Oh, I think I misremembered what I used acpi_call for. I used it to disable the dedicated graphics card on my old Thinkpad, so I haven't tested it on my Dell. But from https://github.com/mkottman/acpi_call it seems to be compatible with other models as well in principle.